src/main/java/mate/academy/spring/boot/model/Book.java

```diff
@@ -32,4 +32,6 @@ public class Book {
     @Column(length = 100)
     private String description;
     private String coverImage;
+    @Column(nullable = false, columnDefinition = "TINYINT(1)")
```

Comment:
```suggestion
```

Reply:
But if i delete this anno @Column(nullable = false, columnDefinition = "TINYINT(1)") - my app has not start: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: wrong column type encountered in column [is_deleted] in table [books]; found [tinyint (Types#TINYINT)], but expecting [bit (Types#BOOLEAN)]

Reply:
That's why I suggested to changed 01-create-books-table.yaml `type: BIT ----> type: boolean` also (see comment in changelog file)

Reply:
In this comit i fixed all problem : https://github.com/Chorniypistolet/spring-boot/pull/5/commits/37c95447c32f40dad82cb5390a70ba2c48964f49

Reply:
You still use `@Data` for entity
Read [article](https://medium.com/@miguelangelperezdiaz444/the-hidden-dangers-of-lombok-annotations-in-your-java-code-what-you-need-to-know-8acdce2d6b89)  about it and replace to necessary annotations

Reply:
Also, you can squash commits to one so we can see the latest version of you changes